### PR TITLE
dockerize the build of genesis live CDs

### PR DESCRIPTION
**Pull request by byxorna**
@tumblr/systems @roymarantz @defect @Primer42 @maddalab This will make building genesis live CDs easier, by dockerizing the dependencies.

### CODE REVIEW

**Comment by byxorna** [comment]
Of note: there is an issue where a newer version (SL7's version) of livecd creator will fail to build genesis, complaining `Error creating Live CD : Unable to run ['/usr/bin/firewall-offline-cmd', '--disabled']!`.

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+export GENESIS_DIR="${GENESIR_DIR:-/genesis}"
+export OUTPUT_DIR="${OUTPUT_DIR:-/output}"
+
+set -ex
+echo "Performing build of Genesis Framework"
+cd $GENESIS_DIR
+
```
Comment (roymarantz) [inline docker-entrypoint.sh]: what about building the supporting gems in src?  i.e.  framework, retryingfetcher, and prompt-cli

Comment (byxorna) [inline docker-entrypoint.sh]: Why? https://github.com/tumblr/genesis/blob/master/bootcd/genesis.ks#L205-L209 just installs them from rubygems, so locally built gems dont matter.

Comment (roymarantz) [inline docker-entrypoint.sh]: Interesting, I didn't think we published them to rubygems, just our local gem server.

Comment (byxorna) [inline docker-entrypoint.sh]: https://rubygems.org/gems/genesis_promptcli https://rubygems.org/gems/genesis_framework https://rubygems.org/gems/genesis_retryingfetcher

**Comment by Primer42** [comment]
LGTM

**Comment by byxorna** [comment]
@Primer42 have you tested this? I would feel better with someone else running thru my docs and ensuring a bootable image was created. Also, hello!

**Comment by Primer42** [comment]
@byxorna - I tested that it built the images. I haven't tested that the images work yet. I was going to do that next.

**Comment by Primer42** [comment]
Also, hello!

**Comment by byxorna** [comment]
@Primer42 can you please merge this?

**Comment by Primer42** [comment]
Sorry for the delay @byxorna - I'll have automated builds up on the docker hub soon too.

**Pull request merged**